.sequelizerc

```diff
@@ -0,0 +1,5 @@
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('config', 'config.js')
+}
```

Comment:
you can add migration seeds and models path too 

for example this 
`var path = require('path')

module.exports = {
  'config':          path.resolve('src', 'config', 'config.js'),
  'migrations-path': path.resolve('src', 'migrations'),
  'models-path':     path.resolve('src', 'models'),
  'seeders-path':    path.resolve('src', 'seeders'),
}`